webapp/security/helpers.py

```diff
@@ -15,94 +15,90 @@ def get_summarized_status(cve, ignored_low_indicators, vulnerable_indicators):
         Check if all statuses for all packages are the same,
         excluding DNE and empty data
         """
-        if (
-            len(
-                {
-                    d["status"]
-                    for d in package["statuses"]
-                    if d["status"] not in {"DNE", ""}
-                }
-            )
-            == 1
-        ):
-            cve["summarized_status"] = {
-                "name": cve["packages"][0]["statuses"][0]["status"]
+        # if (
+        #     len(
+        #         {
+        #             d["status"]
+        #             for d in package["statuses"]
+        #             if d["status"] not in {"DNE", ""}
+        #         }
+        #     )
+        #     == 1
+        # ):
+        #     cve["summarized_status"] = {
+        #         "name": cve["packages"][0]["statuses"][0]["status"]
+        #     }
+        # else:
+        """
```

Comment:
This is causing a bug in the summarized statuses. Will address in separate pr, commented out for now. The rest of this file is untouched since last reviewed save for indentation

Reply:
Can you please a TODO comment there as well then, to make it clear that you are going to address it afterwards.